test/kwc-feature-list_test.html

```diff
@@ -0,0 +1,89 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+        <title>kwc-feature-list test</title>
+
+        <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+        <script src="../../web-component-tester/browser.js"></script>
+
+        <link rel="import" href="../kwc-feature-list.html">
+    </head>
+    <body>
+        <test-fixture id="basic">
+            <template>
+                <kwc-feature-list></kwc-feature-list>
+            </template>
+        </test-fixture>
+
+        <script>
+            suite('kwc-feature-list', function() {
+                var element;
+
+                setup(function () {
+                    element = fixture('basic');
+                });
+
+                test('instantiating the element works', function() {
+                    assert.equal(element.is, 'kwc-feature-list');
+                });
+
+                test('the title is undefined by default', function() {
+                    assert.equal(element.title, undefined);
```

Comment:
Should we normalise `undefined` properties to `null` properties? Just to make difference between properties that shouldn't exist and properties that do exist but are started without value.